src/snippets/responsive-bg-image.liquid

```diff
@@ -0,0 +1,39 @@
+{% comment %}
+    It creates a style tag containing "background-image" properties to load the correct image depending on the resolution
+    Dependencies:
+    - Lazysizes plugin (https://github.com/aFarkas/lazysizes) which enable responsive image with faster load time and better performance.
+    - Lazysizes Bgset (https://github.com/aFarkas/lazysizes/tree/gh-pages/plugins/bgset) To use responsive images on background-image (CSS)
+
+    Accepts:
+    - image: {Object} Image object
+
+    Usage:
+    In your liquid template file, copy the following line
+
+    <div class="responsive-bg-image lazyload" data-bgset="{% include 'responsive-bg-image', image: article.image %}"></div>
+
+    - It must include "responsive-bg-image lazyload" wrapper element
+
+    Questions:
```

Comment:
Could you move these questions to the PR thread, if you are still looking for answers? We won't be shipping with these 😄

Reply:
Yes, I put on the the thread, thanks!